extensions/2.0/Vendor/DRACO_animation_compression/README.md

```diff
@@ -0,0 +1,236 @@
+
+```javascript
+{
+    "extensionsRequired" : [
+        "Draco_animation_compression"
+    ],
+    "extensionsUsed" : [
+        "Draco_animation_compression"
+    ],
+    "extensions" : {
+        "Draco_animation_compression" : {
+            "compressedAnimations" : [
+                {
+                    "input" : 0,
+                    "outputs" : [ 1, 2, 3 ],
+                    "attributeId" : [ 1, 2, 3 ],
+                    "bufferView" : 0
+                },
+                {
+                    "input" : 4,
+                    "outputs" : [ 5 ],
+                    "attributeId" : [ 1 ],
+                    "bufferView" : 1
+                }
+            ],
+        }
+    },
+    "nodes" : [
+      ...
+    ],
+    "animations" : [
+        {
+            "channels" : [
+                {
+                    "sampler" : 0,
+                    "target" : {
+                        "node" : 0,
+                        "path" : "translation"
+                    }
+                },
+                {
+                    "sampler" : 1,
+                    "target" : {
+                        "node" : 1,
+                        "path" : "rotation"
+                    }
+                },
+            ],
+            "samplers" : [
+                {
+                    "input" : 0
+                    "interpolation" : "LINEAR",
+                    "output" : 1
+                },
+                {
+                    "input" : 0
+                    "interpolation" : "LINEAR",
+                    "output" : 2
+                }
+            ]
+        },
+        {
+            "channels" : [
+                {
+                    "sampler" : 0,
+                    "target" : {
+                        "node" : 2,
+                        "path" : "rotation"
+                    }
+                },
+                {
+                    "sampler" : 1,
+                    "target" : {
+                        "node" : 3,
+                        "path" : "translation"
+                    }
+                },
+            ],
+            "samplers" : [
+                {
+                    "input" : 0
+                    "interpolation" : "LINEAR",
+                    "output" : 3
+                },
+                {
+                    "input" : 4
+                    "interpolation" : "LINEAR",
+                    "output" : 5
+                }
+            ]
+        },
+    ],
+    "accessors" : [
+        // Accessors of input/output of animations that use the compression extension will omit "bufferView" property.
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "SCALAR"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC3"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC4"
```

Comment:
Do all properties of the original accessor (componentType, count, ...) remain valid for the compressed data?

Reply:
Yes